### ISSUE TITLE
Missing symbol type for newer zio-aws versions

### ISSUE THREAD

**Issue by joroKr21** [zio/zio-sqs#599]
`zio-sqs` with newer version of `zio-aws` (e.g. 3.17.100.3)
```
Symbol 'type io.github.vigoo.zioaws.sqs.model.Message.ReadOnly' is missing from the classpath.
[error] This symbol is required by 'value zio.sqs.SqsStream.msg'.
[error] Make sure that type ReadOnly is in your classpath and check for conflicting dependencies with `-Ylog-classpath`.
[error] A full rebuild may help if 'SqsStream.class' was compiled against an incompatible version of io.github.vigoo.zioaws.sqs.model.Message.
```

**Comment by ghostdogpr** [comment]
It was updated yesterday in https://github.com/zio/zio-sqs/pull/598
There is a snapshot you can use https://oss.sonatype.org/content/repositories/snapshots/dev/zio/zio-sqs_2.13/0.4.2+65-f4104edd-SNAPSHOT/
Do you need an official release?

**Comment by joroKr21** [comment]
> It was updated yesterday in #598

Oh nice thanks 👍 - I've been seeing this error for a while and finally decided to open an issue 😄 

> Do you need an official release?

That would be nice yeah 🙏

**Comment by ghostdogpr** [comment]
Just released 0.4.3!
